pkg/log/format.go

```diff
@@ -1,73 +1,75 @@
 package log
 
 import (
-	"bytes"
-	"fmt"
+	"log/slog"
+	"os"
 	"sort"
-	"strings"
 	"time"
 
 	"github.com/fatih/color"
-	"github.com/sirupsen/logrus"
 )
 
-func getGCPFormatter() logrus.Formatter {
-	return &logrus.JSONFormatter{
-		FieldMap: logrus.FieldMap{
-			logrus.FieldKeyLevel: "severity",
-			logrus.FieldKeyMsg:   "message",
-			logrus.FieldKeyTime:  "timestamp",
```

Comment:
We need to also replace attrs for these as well as the level in order for the logs to be respected in GCP environments.

Reply:
ok, but anything more? I would love a detailed review.

Reply:
will work on this

Reply:
have added code for this please check and let me know if this is good enough